fili-core/src/main/java/com/yahoo/bard/webservice/data/metric/protocol/ProtocolSupport.java

```diff
@@ -0,0 +1,148 @@
+// Copyright 2020 Oath Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.data.metric.protocol;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * An object to describe which protocols a metric can support and to invoke the apply for that protocol.
+ *
+ * In the case where a stack of metrics might be delegated to, white or black listing allows the handler to indicate
+ * whether delegation should be allowed.
+ */
+public class ProtocolSupport {
+
+    /**
+     * A substitute for Boolean to indicate Yes, No or Uknown in the case where delegates might service a signal.
+     */
+    public enum Accepts {
+        TRUE,
+        FALSE,
+        MAYBE  // If this Handler cannot accept the protocol, but doesn't make positive claims if another might.
```

Comment:
I stumbled a little here because initially I expected FALSE to mean "I
don't accept this Protocol, but a different handler might." As opposed to "You
can have my metric when you pry it from my cold, dead fingers." I worry that this
might be similarly confusing for others, and a source for bugs. One option is to
line things up with how *I* envisioned it, and do something like TRUE, FALSE, REJECT.
But I'm not convinced that helps so much as change the set of people who will be confused.
I wonder if maybe better names? Something that more emphatically states that FALSE should
be used when you want to stop processing on the metric? Something like "ACCEPT", "REJECT",
and "MAYBE"?